src/commands/service/stop.ts

```diff
@@ -28,15 +28,14 @@ export default class ServiceStop extends Command {
   async run(): Promise<string[]> {
     const {argv, flags} = this.parse(ServiceStop)
     if (flags['delete-data'] && !flags.confirm) {
-      cli.warn('This will delete all data associated to this instance')
+      cli.warn('This will delete all the data associated with this running service')
       if (!await cli.confirm('Are you sure?')) return []
     }
-    this.spinner.start('Delete instance')
+    this.spinner.start('Stop running services')
     for (const hash of argv) {
-      this.spinner.status = hash
       await this.api.instance.delete({hash, deleteData: flags['delete-data']})
     }
-    this.spinner.stop(argv.join(', '))
+    this.spinner.stop()
```

Comment:
```suggestion
    this.spinner.stop(argv.join(', '))
```
we need to keep important data in spinner for now